server/service.go

```diff
@@ -200,6 +206,22 @@ func (m *BoostService) sendValidatorRegistrationsToRelayMonitors(payload []types
 	}
 }
 
+func (m *BoostService) sendAuctionTranscriptToRelayMonitors(transcript AuctionTranscript) {
```

Comment:
nitpick but we could pass in a pointer here
```suggestion
func (m *BoostService) sendAuctionTranscriptToRelayMonitors(transcript *AuctionTranscript) {
```

I tend to do so unless it is clear the input will *always* be small enough

Reply:
+1